test/Container/NotFoundMiddlewareFactoryTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
2018 only, please

Reply:
This file was renamed from `NotFoundHandler`; date range is correct.

Reply:
Did you use `git mv` to rename the file? We are loosing the connection between new and old file, it looks like new file...
Shouldn't we update year range as the file is updated (renamed)?

Reply:
best way is to move file 100% unchanged, and then edit it in another commit as git does not really have "move" type of change. Simple diff might not detect it anyway as it uses similarity threshold. It is more for a 3-way merge benefit.

Reply:
... or use PhpStorm :D 

It keeps track of moved files (most of the time) so you can move and edit it in the same commit. Not sure how to do that in git. But as PhpStorm can do it, it must be doable with git as well.

Reply:
I did a `git mv`, but then also added the new file in the same commit, which is why this happened.

Note to self: be more granular in commits.